api/nyt.py

```diff
@@ -76,20 +83,31 @@ def from_config(cls, _db, **kwargs):
             message = "No ExternalIntegration found for the NYT."
             raise CannotLoadConfiguration(message)
 
-        if not integration.password:
-            message = "NYT integration improperly configured."
-            raise CannotLoadConfiguration(message)
-
         return cls(_db, api_key=integration.password, **kwargs)
 
     def __init__(self, _db, api_key=None, do_get=None, metadata_client=None):
+        self.log = logging.getLogger("NYT API")
         self._db = _db
+        if not api_key:
+            raise CannotLoadConfiguration("No NYT API key is specified")
         self.api_key = api_key
         self.do_get = do_get or Representation.simple_http_get
         if not metadata_client:
-            metadata_client = MetadataWranglerOPDSLookup.from_config(self._db)
+            try:
+                metadata_client = MetadataWranglerOPDSLookup.from_config(
+                    self._db
+                )
+            except CannotLoadConfiguration, e:
+                self.log.error(
+                    "Metadata wrangler integration is not configured, proceeding without one."
+                )
```

Comment:
It doesn't look like this needs to be a fatal error -- worst case we'll use the default code to turn "Surname, Name" into "Name Surname".